### PR TITLE
Add Arch Linux installation instructions

### PR DESCRIPTION
**Pull request by agura-lex**
**Description**
Add the instruction for AUR installation and some notes about `acpi_call` package in Arch.

**Additional context**
Discussed in https://github.com/dominiksalvet/asus-fan-control/issues/11

### CODE REVIEW

**Comment by agura-lex** [comment]
Tried to keep it consistent with everything else, but will likely need some edits

**Comment by dominiksalvet** [comment]
Well, from your comments, I assume that Arch users are able to work with AUR. For me, *README.md* seems cluttered as per the current state of this PR changes. What do you think about reducing it to necessary minimum, I mean something like "Hey, asus-fan-control is also in AUR..."?

Currently, I am also thinking about rewriting/rearranging the whole *README.md* as it starts to be exhaustive and it may discourage some users. This is a simple program, so its description should be as well...

**Comment by agura-lex** [comment]
Yeah, seems like a good idea, probably will shorten it to that

**Comment by agura-lex** [comment]
I think it looks good enough now

File: README.md [hunk 1]
```diff
@@ -42,15 +44,28 @@ It appears that this project could work with a **whole range of ASUS notebook mo
 * **systemd** to start it at boot
 * **acpi_call** kernel module
   * If your system has `apt-get`, it will be installed automatically.
+  * If you're using Arch Linux with a *non-standard* kernel, be sure to have `acpi_call` version for your kernel (`acpi_call-lts` for the LTS kernel and `acpi_call-dkms` for everything else)
```
Comment (dominiksalvet) [inline README.md]: Is this note important?

Comment (agura-lex) [reply README.md]: I believe it is. But:
- a user should get a note about [optional dependencies](https://aur.archlinux.org/cgit/aur.git/commit/?h=asus-fan-control&id=1cd3b53472e7c6fdbe3d74347b417b91d70082f6) when installing asus-fan-control, which might be informative enough. 
- my [post-install](https://aur.archlinux.org/cgit/aur.git/tree/asus-fan-control.install?h=asus-fan-control) script is also redundantly informative

So maybe we can remove it. ¯\\_(ツ)_/¯

Comment (dominiksalvet) [reply README.md]: I don't want to create unnecessary *README.md* clutter any further, so feel free to remove it from the PR.

Comment (agura-lex) [reply README.md]: Done

File: README.md [hunk 2]
```diff
@@ -45,12 +47,24 @@ It appears that this project could work with a **whole range of ASUS notebook mo
...
 
+#### From AUR (for Arch-based distros)
+
+If you're using Arch Linux or Arch-based distribution, there's an [AUR package](https://aur.archlinux.org/packages/asus-fan-control) available, just [install](https://wiki.archlinux.org/index.php/Arch_User_Repository#Installing_packages) `asus-fan-control` from AUR. 
+
+The package doesn't automatically enable starting the service at boot (since Arch packages generally don't do that), to enable it at boot:
+
+```sh
+sudo systemctl enable asus-fan-control
+```
```
Comment (dominiksalvet) [inline README.md]: BTW, is this really needed? Based on your comment, I understand that AUR packages generally don't enabled a service at the boot. But, I would like to have both GitPack and AUR installations consistent and hence having it enabled at boot would be required. Furthermore, I do not see any point of installing asus-fan-control on systemd-based system and not having it set up to run at boot...

So would it be possible to include changes that cause that after asus-fan-control installation it is set up to launch at boot, please? Or do you have any argument against it, except the provided one?

Comment (agura-lex) [reply README.md]: It's not just AUR pacakges, it's Arch packages in general, they usually don't enable services unless it's some vital part of the system, so I'm keeping it consistent with the distribution.

Comment (dominiksalvet) [reply README.md]: OK, I respect that. However, I am going to remaster *README.md* in the future. It is possible that this particular paragraph will be simplified but do not worry - no key information will be lost.

**Comment by dominiksalvet** [comment]
Thank you so much for the contribution and patience with me! :heart:

**Pull request merged**